WebApi/Services/TicketPdfServiceQuestPdf.cs

```diff
@@ -0,0 +1,134 @@
+﻿using QRCoder;
+using QuestPDF.Fluent;
+using QuestPDF.Helpers;
+using QuestPDF.Infrastructure;
+using WebApi.Interfaces.Services;
+using WebApi.Models;
+
+namespace WebApi.Services
+{
+    public class TicketPdfServiceQuestPdf : ITicketPdfService
+    {
+        private readonly byte[] headerImageBytes;
+        private readonly QRCodeGenerator qRCodeGenerator;
+
+        public TicketPdfServiceQuestPdf()
+        {
+            try
+            {
+                headerImageBytes = File.ReadAllBytes("Resources/Images/Cinemagia_logo.jpg");
+            } catch(Exception)
+            {
+                headerImageBytes = Placeholders.Image(10, 10);
+            }
```

Comment:
```suggestion
            headerImageBytes = File.Exists("Resources/Images/Cinemagia_logo.jpg") ? File.ReadAllBytes("Resources/Images/Cinemagia_logo.jpg") : Placeholders.Image(10, 10);
```

Dit kan een one liner worden, sinds je de Exception van de try/catch toch niet gebruikt is het onnodig.